CodeFusion/README.md

```diff
@@ -0,0 +1,28 @@
+# CodeFusion Data and Benchmarks
+
+This repository contains the conditional formatting benchmarks used in CodeFusion (CodeFusion: A Pre-trained Diffusion Model for Code Generation)
+
+## Folder Structure
+
+- `Benchmarks`
+    - Training and Testing data for Python, M and CF Rules
+- `Pretraining Data`
+    - Unannotated code used for pretraining for Python, M and CF Rules
```

Comment:
right now I only see Bash. We should amend this. If we are not releasing CF due to bing21 issues, then let's just state we cannot due to compliance.

Reply:
Added now.